packages/rancher-istio/generated-changes/dependencies/kiali/dependency.yaml

```diff
@@ -1,2 +1,2 @@
 workingDir: ""
-url: packages/rancher-kiali-server
+url: packages/rancher-kiali-server-1.35
```

Comment:
just change this URL to point to the released assert. dependency.yaml supports everything package.yaml supports, so you can specify it as a release tgz or a git repository (pointing to release-v2.5, with subdirectory and commit).

This way we can avoid having an rancher-kiali-server release or a rancher-kiali-server-crd release. Also, no need to change annotations.

Reply:
awesome! I have redone the PR to use this suggestion!